src/libsyntax/diagnostic_list.rs

```diff
@@ -16,31 +16,392 @@
+#[inline]
+fn something() {}
+```
+
+For more information about the inline attribute, read:
+https://doc.rust-lang.org/reference.html#inline-attributes
+"##,
+
+E0535: r##"
+An unknown argument was given to the `inline` attribute.
+
+Erroneous code example:
+
+```compile_fail,E0535
+#[inline(unknown)] // error: invalid argument
 pub fn something() {}
 
 fn main() {}
 ```
+
+The `inline` attribute only supports two arguments:
+
+ * always
+ * never
+
+All other arguments given to the `inline` attribute will return this error.
+Example:
+
+```
+#[inline(never)] // ok!
+pub fn something() {}
+
+fn main() {}
+```
+
+For more information about the inline attribute, https:
+read://doc.rust-lang.org/reference.html#inline-attributes
+"##,
+
+E0536: r##"
+The `not` cfg-predicate was malformed.
+
+Erroneous code example:
+
+```compile_fail,E0536
+#[cfg(not())] // error: expected 1 cfg-pattern
+pub fn something() {}
+
+fn main() {}
+```
+
+The `not` predicate expects one cfg-pattern. Example:
+
+```
+#[cfg(not(target_os = "linux"))] // ok!
+pub fn something() {}
+
+fn main() {}
+```
+
+For more information about the cfg attribute, read:
+https://doc.rust-lang.org/reference.html#conditional-compilation
+"##,
+
+E0537: r##"
+An unknown predicate was used inside the `cfg` attribute.
+
+Erroneous code example:
+
+```compile_fail,E0537
+#[cfg(unknown())] // error: invalid predicate `unknown`
+pub fn something() {}
+
+fn main() {}
+```
+
+The `cfg` attribute supports only three kinds of predicates:
+
+ * any
+ * all
+ * not
+
+Example:
+
+```
+#[cfg(not(target_os = "linux"))] // ok!
+pub fn something() {}
+
+fn main() {}
+```
+
+For more information about the cfg attribute, read:
+https://doc.rust-lang.org/reference.html#conditional-compilation
+"##,
+
+E0538: r##"
+Multiple conflicting stability annotations for the same item.
+
+Erroneous code example:
+
+```compile_fail
+#[unstable(feature = "some_feature", issue = "1")]
+#[unstable(feature = "another_feature", issue = "1")]
+// error: multiple stability items
+pub fn something() {}
+```
+
+Item may have only one of each stability annotations applied to it. Example:
+
+```ignore
+#[unstable(feature = "another_feature", issue = "1")] // ok!
+pub fn something() {}
+```
+"##,
+
+E0539: r##"
+A named attribute parameter doesn't have a value.
+
+Erroneous code example:
+
+```compile_fail
+#[stable(feature)] // error: incorrect meta item
+pub fn something() {}
+```
+
+Named attribute parameters need a value:
+
+```ignore
+#[stable(feature = "some_feature")] // error: incorrect meta item
+pub fn something() {}
+```
+"##,
+
+E0540: r##"
+Multiple conflicting `rustc_deprecated` annotations for the same item.
+
+Erroneous code example:
+
+```compile_fail
+#[rustc_deprecated(since = "1.3.0", reason = "some reason")]
+#[rustc_deprecated(since = "1.4.0", reason = "no reason")]
+// error: multiple rustc_deprecated attributes
+pub fn something() {}
+```
+
+For obvious reason, an element can only be deprecated once. Example:
```

Comment:
This statement seems patronizing

"An item can only have one stability level." (from later on in the PR) is a good statement, so I'd go with "An item can only be deprecated once."

Also, is there a difference between an item and an element? I've never seen the term "element" w.r.t. Rust, so I switched it to item.